### ISSUE TITLE
Allow to consume "get" methods in requirements matchers

### ISSUE THREAD

**Issue by apodhrad** [eclipse/reddeer#1786]
```
new RequirementMatcher(Server.class, "server.class", new FooMatcher())
```
where FooMatcher will match
```
server.getClass()
```

**Comment by rawagner** [comment]
it would be better to use "server.getClass()"

**Comment by apodhrad** [comment]
Only standard property access syntax will be supported, e.g. "server.class".

**Comment by apodhrad** [comment]
fixed as a part of #1785
